lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -355,6 +351,35 @@ export class ServerRequestParameters {
         return paramsString;
     }
 
+    /**
+     * @hidden
+     *
+     * Validate  scopes/state/correlationId set in the request by the user
+     * @param request
+     */
+    static validateUserSetServerParams(scopes: Array<string>, state: string, correlationId: string, clientId: string): UserSetRequestParams {
+
+        // set scope to clientId if null
+        let reqScopes: Array<string>;
+        if (!scopes) {
+            reqScopes = [clientId];
+        } else {
+            reqScopes = [ ...scopes ];
```

Comment:
This can be a ternary instead (I know its copy pasta, but its chance to make it better :) )

Reply:
Yes :)

Reply:
Done.